runtime/doc/news.txt

```diff
@@ -148,7 +148,8 @@ OPTIONS
 
 PLUGINS
 
-• TODO
+• 'inccomand' incremental preview now runs on 'nomodifiable' buffers and
+  restores their 'modifiable' state
```

Comment:
this is in news-breaking. please find the PLUGINS section in news-features, below :)